ethers-core/src/types/trace/geth.rs

```diff
@@ -87,3 +138,14 @@ where
 {
     s.serialize_str(&hex::encode(x.as_ref()))
 }
+
+fn serialize_bytes_opt<S, T>(x: &Option<T>, s: S) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+    T: AsRef<[u8]>,
+{
+    match x {
+        Some(x) => serialize_bytes(x, s),
+        None => s.serialize_none(),
+    }
+}
```

Comment:
this and the existing `serialize_bytes` function seem unnecessary because `Bytes` already does that I think

or do they differ? if so needs docs.